tokens/finn.no/tabs.yml

```diff
@@ -5,13 +5,13 @@ color:
   tabs:
     border:
       _: transparent
-      hover: gray-200
-      active: blue-600
+      hover: s-color-border-primary
```

Comment:
We've discussed this already, too, but wanted to mention it here for future ref (also commented in the [spreadsheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1588196525&range=E384)).

Now both hover and selected state apply the same border color, which doesn't seem intentional:
![Screenshot 2023-07-11 at 09 07 29](https://github.com/warp-ds/css/assets/41303231/27927e8b-d8b0-41ae-9bef-73fe52ff7a92)